### ISSUE TITLE
Multiple instances of the same Sound

### ISSUE THREAD

**Issue by jackdaus** [StereoKit/StereoKit#424]
## Describe the feature
It would be great if we could play the same `Sound` in two different locations at once. As far as I can tell from the documentation, this currently doesn't seem to be possible.

## An example use-case
I'm imagining we might have two models that periodically make use of the same `Sound`. The timing of the two instances of the `Sound` might overlap. For example, if we have two bees buzzing:

```
		public SKSettings Settings => new SKSettings { 
			appName           = "StereoKit Template",
			assetsFolder      = "Assets",
			displayPreference = DisplayMode.MixedReality
		};

		Pose  pose1 = new Pose(1, 0, 0, Quat.Identity);
		Model bee1;
		Sound buzz1;

		Pose pose2 = new Pose(-1, 0, 0, Quat.Identity);
		Model bee2;
		Sound buzz2;

		public void Init()
		{
			bee1	= Model.FromFile("bee_alt2.glb");
			buzz1	= Sound.FromFile("bee_alt.mp3");
			buzz1.Play(pose1.position);

			// The sound from buzz2 stomps the sound from buzz1
			bee2	= Model.FromFile("bee_alt2.glb").Copy();
			buzz2	= Sound.FromFile("bee_alt.mp3");
			buzz2.Play(pose2.position);
		}

		public void Step()
		{
			bee1.Draw(pose1.ToMatrix(0.01f));
			bee2.Draw(pose2.ToMatrix(0.01f));
		}
	}
```

The sound from buzz1 gets stomped by the sound from buzz2. It seems that buzz1 or buzz2 cannot both play at the same time. 

Here are the Asset files in case you're interested in playing with this particular use case:
[bee_media.zip](https://github.com/StereoKit/StereoKit/files/9060086/bee_media.zip)

**Comment by maluoi** [comment]
Yeah, that's definitely a limitation currently! I just did a bit of investigation into how much effort it'd take to change that, and didn't see a really quick way to add this feature. A bit more investigation might reveal a nice path, but I think this is one I'll have to backburner for a bit! v0.3.7 is looking a little full for me right now 😬

---

_Just some notes here for future me regarding what I found while investigating:_

StereoKit currently uses `ma_decoder` to represent the audio data, which represents both the audio data _and_ the position in the audio file. It seems like it can only be at one location in the audio file at a time, so something extra is needed here.

Miniaudio apparently [has a high level API](https://miniaud.io/docs/manual/index.html#HighLevelAPI) I somehow missed. This could either be good reference for figuring out a way to play an audio file multiple times, or might be an interesting thing to just switch over to entirely! It comes with some more [advanced spatialization](https://miniaud.io/docs/manual/index.html#Engine) features.

**Comment by briantcluff** [comment]
Similarly, being able to pause, resume, and seek sound would be a nice feature.
